src/app/ui/persmission-handler.js

```diff
@@ -143,8 +143,8 @@ export class PermissionHandler {
       ? yo`<input type="checkbox" onchange="${switchMode}" checkbox class="custom-control-input" id="remember">`
       : yo`<input type="checkbox" onchange="${switchMode}" class="custom-control-input" id="remember">`
     const message = remember
-      ? `${fromName} has changed and would like to access the plugin ${toName}.`
-      : `${fromName} would like to access plugin ${toName}.`
+      ? `${fromName} has changed and would like to access ${toName}.`
+      : `${fromName} would like to access ${toName}.`
```

Comment:
With your change it reads,"FLATTENER WOULD LIKE TO ACCESS PLUGIN FILE MANAGER. :"  
So we should remove the "." or the ":" or both.   Because it is a headline - I think you should remove both. But either way ( just not both)

Reply:
oh ok I will remove '.'

Reply:
it is still a bit not smooth. maybe "Flattener" would like to access "File Manager" plugin?

Reply:
I don't want to use the word plugin here. do you like to have double quote?

Reply:
then at least double quotes, yes.